dash_docs/chapters/dash_enterprise/dash_enterprise_chapters.py

```diff
@@ -695,235 +695,1654 @@ def display_instructions_deploy(method):
+
+                ***
+
+                // MARK: Procfile Python
+
+                #### Procfile
+
+                `Procfile` is a required text file that declares which commands are run by your Dash app on
+                startup like starting your app's web server, scheduling jobs and running background processes.
+
+                This file is always named `Procfile` — with no file extension. It must be placed in your app's root
+                directory and uses the following format:
+
+                ```
+                <process type>: <command>
+
+                ```
+
+                Process type names are arbitrary, and can be anything, however there are 
+                two special process types:
+
+                1. `web`
+                2. `release`
+                
+                
+                A simple Dash app `Procfile` will resemble the following:
+
+                ```
+                web: gunicorn app:server --workers 4
+                ```
+
+                A Dash app running  a background task queue will have `Procfile` similar to this:
+
+                ```
+                web: gunicorn app:server --workers 4
+                worker: celery -A app:celery_instance worker
+                ```
+
+                A Dash app running generating report with the Snapshot Engine will have `Procfile` like this:
+
+                ```
+                web: gunicorn index:server --workers 4
+                worker: celery -A index:celery_instance worker --concurrency=2
+                scheduler: celery -A index:celery_instance beat
+                ```
+                Note that `worker` and `scheduler` process names are arbitrary. 
+                We recommend using desciptive names as they will appear in all logs  and in the 
+                app manager UI.
+
+                ##### web process
+
+                `web` is the only process type that can receive external HTTP traffic. We use
+                it to run `gunicorn`, your Dash app's web server. In the following example we are
+                declaring `gunicorn` as our web server with `web: gunicorn`:
+
+                ```
+                web: gunicorn app:server --workers 4
+
+                ```
+
+                Here we have `app` that refers to the file `app.py` and `server` refers to the
+                variable named `server` inside that file. `gunicorn` is the web server that will
+                run your Dash app, make sure to add this in your `requirements.txt` file.
+
+                If the entrypoint of your app has a different file name like index.py the the 
+                `Procfile` will look like: 
+                
+                ```
+                web: gunicorn index:server --workers 4
+                ```
+        
 
-    ## Folder Reference
+                With `gunicorn` as your app's web server, you may also use the following flags:
```

Comment:
```suggestion
```suggestion
        These commands are [standard `gunicorn` commands](https://docs.gunicorn.org/en/latest/run.html).
        
        `gunicorn` accepts a [wide variety of settings](https://docs.gunicorn.org/en/latest/settings.html). 
        Here are a few common flags:
```
```